cqrs-example/cqrs-example.cabal

```diff
@@ -24,12 +24,12 @@ Executable cqrs-example
   Main-is:           CQRSExample/Main.hs
   Ghc-options:       -Wall
   Build-depends:     base >= 4.8 && < 5
-                   , aeson >= 0.8 && < 0.9
+                   , aeson >= 0.8
```

Comment:
Versions in stackage `lts-5.2` are:

    aeson-0.9.0.1
    cereal-0.5.1.0
    warp-3.2.2

That is no longer the latest stackage lts, but I just checked, and `lts-5.16` has the same versions for those packages.